neuronav/envs/grid_env.py

```diff
@@ -158,10 +159,22 @@ def reset(
         else:
             self.agent_pos = self.agent_start_pos
 
+        # if both objects and reward_locs are None, throw an error
+        if objects == None and reward_locs == None:
```

Comment:
This should actually be fine, as in this case the behavior should be to do: `self.objects = self.topo_objects`